roq-deployment/development/GCM.log

```diff
@@ -0,0 +1,9 @@
+Starting the  global configuration manager
+10:34:42,660 ERROR [main] (FileConfigurationReader:loadGCMConfiguration:53)   Error while reading configuration file - skipped but set the default configuration
+java.util.NoSuchElementException: 'hasCloudConfiguration' doesn't map to an existing object
+	at org.apache.commons.configuration.AbstractConfiguration.getBoolean(AbstractConfiguration.java:644)
+	at org.roqmessaging.management.config.internal.FileConfigurationReader.loadGCMConfiguration(FileConfigurationReader.java:47)
+	at org.roqmessaging.management.GlobalConfigurationManager.<init>(GlobalConfigurationManager.java:96)
+	at org.roqmessaging.management.launcher.GlobalConfigurationLauncher.main(GlobalConfigurationLauncher.java:43)
```

Comment:
This exception indicates a problem in the code when reading a gcm properties file. Was this problem fixed?